client/components/NavBar.jsx

```diff
@@ -14,18 +15,32 @@ function NavBar (props) {
     useEffect(() => {
         if (cookie.token) {
             setLoggedIn(true);
-            console.log("Logged in")
         } else {
             setLoggedIn(false);
-            console.log("Logged out")
         }
     }, [cookie.token]);
 
+    async function myLogOut() {
```

Comment:
Could you name it "doLogOut" instead? "myLogOut" seems a little amateurish.
Also, could you change it to a const anonymous function, like so:
```js
const doLogOut = async () => {
...
}
```
This fits more with the style of making const anonymous functions for callbacks.